ndb/src/google/cloud/ndb/blobstore.py

```diff
@@ -56,10 +57,16 @@
 ]
 
 
+BlobKey = _ports.BlobKey
```

Comment:
To follow the winding path in the original:

1. https://github.com/GoogleCloudPlatform/datastore-ndb-python/blob/cf4cab3f1f69cd04e1a9229871be466b53729f3f/ndb/blobstore.py#L107
1. https://github.com/GoogleCloudPlatform/datastore-ndb-python/blob/cf4cab3f1f69cd04e1a9229871be466b53729f3f/ndb/google_imports.py#L67
1. https://github.com/dhermes/app-engine-standard/blob/19187b658bd11bac1edb0f203c22a14444fc21ff/google/appengine/api/blobstore/__init__.py#L23
1. https://github.com/dhermes/app-engine-standard/blob/19187b658bd11bac1edb0f203c22a14444fc21ff/google/appengine/api/blobstore/blobstore.py#L74

In other words, it's the same (`datastore_types.BlobKey`) as the one in `ndb.model`